packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -73,3 +73,8 @@ export const BackdropView = styled.div`
 	${ reduceMotion( 'transition' ) }
 	z-index: 1;
 `;
+
+export const VisualLabelWrapper = styled.div`
+	// Makes the inline label be the correct height, equivalent to setting line-height: 0
+	display: flex;
+`;
```

Comment:
This is a tiny fix we need on the ToggleGroupControl label that causes a 1px height discrepancy with the other components, due to `block`/`inline` differences 😭 We might need to rethink the `BaseControl.VisualLabel` styles if this happens a lot.

https://user-images.githubusercontent.com/555336/185179405-b096ec83-95cf-4053-814a-5a3431376ef4.mp4

Reply:
Does it work if you remove `VisualLabelWrapper` altogether and put `VisualLabel` and `RadioGroup` into a `HStack`? I think that's what I did for `FontSizePicker`.

Reply:
In terms of the label height, yes, anything that makes it a flex child will do. In this case the main RadioGroup defaults to an inline-flex for some reason, so we'll have to look into that more before we can use a cleaner solution like the `HStack` you mention. (Ideally we just use the default BaseControl features for the label rendering, not this custom thing.)